docs/ARM.md

```diff
@@ -238,14 +243,85 @@ Command:
 knife azurerm server create
 --azure-resource-group-name MyResourceGrpName
 --azure-vm-name MyNewVMName
---azure-service-location 'WEST US'
+--azure-service-location 'westus'
 --azure-image-os-type centos
 --azure-vm-size Small
 --server-count 3
 -x myuser -P mypassword
 ```
 This will create 3 VMs with names: `MyNewVMName0`, `MyNewVMName1` and `MyNewVMName2`
 
+### --extended-logs option
```

Comment:
The font of this option is different from `--server-count option`. There should be 4 hash `####`.